src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,320 @@
 [
   {
+    "name": "Google Cloud Speech",
+    "documentation": "https://cloud.google.com/speech/docs/reference/libraries",
+    "description":"Google Cloud Speech API converts audio to text in over 100 languages.",
+    "transports": ["grpc"],
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Cloud Speech API",
+      "language":"java",
+      "site":"https://cloud.google.com/speech/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/index.html?com/google/cloud/speech/v1beta1/package-summary.html",
+      "infotip":"The com.google.cloud.speech.v1 packages",
+      "launchStage" : "alpha",  
+      "packages" : ["com.google.cloud.speech.v1"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-speech",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-speech",
+         "version" : "0.23.0-alpha"
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Resource Manager",
+    "documentation": "https://cloud.google.com/resource-manager/docs/",
+    "description":"The Google Cloud Resource Manager service enables you to programmatically manage hierarchical resource containers such as organizations, folders, and projects.",
+    "transports": ["http"],
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Cloud Resource Manager API",
+      "language":"java",
+      "site":"https://cloud.google.com/speech/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/?com/google/cloud/resourcemanager/package-summary.html",
+      "infotip":"The com.google.cloud.speech.resourcemanager packages",
+      "launchStage" : "alpha",  
+      "packages" : ["com.google.cloud.resourcemanager"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-resourcemanager",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-resourcemanager",
+         "version" : "0.23.0-alpha"
```

Comment:
Should this be version**s**?  how do we encode the BOM data?

Reply:
These entries should be defined in relation to a common BOM: we don't want to mix-and-match between BOMs, right?  The BOM definition could be be external (e.g., encoded in the file-name like `libraries-0.22.json`), or we could make that an outer element if you wanted a single library definition.

Reply:
We could pull this from the google-cloud-java pom.xml though it's an extra lookup.
E.g. https://search.maven.org/remotecontent?filepath=com/google/cloud/google-cloud-pom/0.23.1-alpha/google-cloud-pom-0.23.1-alpha.pom

Reply:
It would be worth having a test that pulls the dependencies for each and ensures there's no conflicting versions.